public/components/visualize/components/security-alerts.tsx

```diff
@@ -1,33 +1,41 @@
 /*
- * Wazuh app - React component for Visualize.
- * Copyright (C) 2015-2021 Wazuh, Inc.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- *
- * Find more information about this on the LICENSE file.
- */
-
+* Wazuh app - React component for Visualize.
+* Copyright (C) 2015-2021 Wazuh, Inc.
+*
+* This program is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+*
+* Find more information about this on the LICENSE file.
+*/
 import React from 'react';
 import { useFilterManager, useQuery, useRefreshAngularDiscover } from '../../common/hooks';
 import { Discover } from '../../common/modules/discover';
-
+import { getFilterWithAuthorizedAgents } from '../../../react-services/filter-authorization-agents';
+import { useSelector } from 'react-redux';
+import { withReduxProvider } from '../../common/hocs';
+import { useAllowedAgents } from '../../common/hooks/useAllowedAgents'
+ 
 export const SecurityAlerts = () => {
-  const [query] = useQuery();
-  const filterManager = useFilterManager();
-  const refreshAngularDiscover = useRefreshAngularDiscover();
-
-  return (
-    <Discover
-      shareFilterManager={filterManager}
-      query={query}
-      initialColumns={["icon", "timestamp", 'rule.mitre.id', 'rule.mitre.tactic', 'rule.description', 'rule.level', 'rule.id']}
-      implicitFilters={[]}
-      initialFilters={[]}
-      updateTotalHits={(total) => { }}
-      refreshAngularDiscover={refreshAngularDiscover}
-      />
-  )
+ const [query] = useQuery();
+ const filterManager = useFilterManager();
+ const copyOfFilterManager = filterManager
```

Comment:
I would say the `copyOfFilterManager` variable is not using. You could remove this statement.

Reply:
Yes, you have all the reason, it was not being used. It has already been removed. Thanks!